src/components/app/SettingsMenu.tsx

```diff
@@ -93,6 +94,11 @@ const SettingsMenu: FC = () => {
         checkIfPowerChanged("Streamer", streamer.power, previousStreamerPowerState);
     }, [amplifier?.power, streamer.power]);
 
+    const showAmplifierPower = amplifier?.power != null
+    const showAmplifierSources = amplifier?.sources != null
```

Comment:
Looks like this needs a quick Prettier pass.

(I sometimes miss running Prettier myself, so no shade, and please feel free to run Prettier on just the PR's changes -- or the whole file if you prefer. At some point I should run Prettier/Black over everything just to pick up any stragglers).

Reply:
Thanks for spotting this! I've run prettier over this file. `StatusScreen.tsx` also looks like it departs from Prettier's style, but not in ways related to this PR... so should probably wait for a wider cleanup.